cmu_graphics/__init__.py

```diff
@@ -1,4 +1,56 @@
+import __main__
+import os
+
+main_directory = os.path.dirname(__main__.__file__)
+main_sibling_filenames = set(os.listdir())
```

Comment:
I think we probably want to use `main_directory` in this call to `os.listdir()`. The docs say: "The first entry in the module search path is the directory that contains the input script". So we do want `main_directory`, not the current working directory.